airflow-core/src/airflow/api_fastapi/execution_api/versions/__init__.py

```diff
@@ -20,9 +20,11 @@
 from cadwyn import HeadVersion, Version, VersionBundle
 
 from airflow.api_fastapi.execution_api.versions.v2025_04_28 import AddRenderedMapIndexField
+from airflow.api_fastapi.execution_api.versions.v2025_05_20 import DowngradeUpstreamMapIndexes
 
 bundle = VersionBundle(
     HeadVersion(),
+    Version("2025-05-20", DowngradeUpstreamMapIndexes),
```

Comment:
I added a week later, maybe needs change?

Reply:
Its fine, we can change it later, before the release